### PR TITLE
Fixed a grammar issue and added another acronym

### PR DESCRIPTION
**Pull request by phistuck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ If you are willing to contribute, open a pull request to complete, update or fil
 * [AMD](/_glossary/AMD.md): a standard defining how to load JavaScript libraries or modules asynchronously.
 * [AngularJS](/_glossary/ANGULARJS.md): a structural framework for dynamic web apps.
 * [Apache Cordova](/_glossary/APACHECORDOVA.md) (formerly **PhoneGap**): a popular mobile application development framework originally created by Nitobi.
-* [Arity](/_glossary/ARITY.md): the number of arguments of a function.
+* [Arity](/_glossary/ARITY.md): the number of arguments of a function.an
```
Comment (KittyGiraudel) [inline README.md]: Woops, what is going on here? :)

File: _data/abbr.yml [hunk 2]
```diff
@@ -22,6 +22,7 @@ JSON: JavaScript Object Notation
 JSON-LD: JSON for Linked Data
 MVC: Model View Controller
 MVVM: Model View ViewModel
+OO: Object-oriented
```
Comment (KittyGiraudel) [inline _data/abbr.yml]: Maybe we could also capitalize oriented?

File: README.md (hunk 1 above)
Comment (phistuck) [inline README.md]: Not sure. Probably happened due to editor slowliness. Fixed.

File: _data/abbr.yml (hunk 2 above)
Comment (phistuck) [inline _data/abbr.yml]: I guess. Awkward, though.

**Comment by KittyGiraudel** [comment]
Awesome, thank you. :)

**Pull request merged**